vsts/publish-podspec.sh

```diff
@@ -66,6 +66,17 @@ if [ "$mode" == "internal" ] || [ "$mode" == "test" ]; then
 
   if [ "$mode" == "test" ]; then
 
+    # Read publish version for current build
+    version=$(cat $CURRENT_BUILD_VERSION_FILENAME)
+
+    # Exit if response doesn't contain an array
```

Comment:
Does it really contain an array, not just the version?

Reply:
Copy-paste. 😄